lint-manifest.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# vim: set ft=sh
+
+set -e
+
+#
+# Validate environment
+#
+if [[ ! -d "pipeline-config" || ! -d "lint-manifest" ]]; then
+  echo "Directories pipeline-config and lint-manifest must exist" >&2
+  exit 1
+fi
+if [ -z "$MANIFEST_FILE" ]; then
+  echo "You must specify \$MANIFEST_FILE" >&2
+  exit 1
+fi
+if [ -z $(which bosh-lint) ]; then
+  echo "bosh-lint not found" >&2
+  exit 1
+fi
+
+#
+# lint the manifest, using a $LINTER_CONFIG if provided
+#
+if [ -f "pipeline-config/${LINTER_CONFIG}" ]; then
```

Comment:
You should probably error if they specify `LINTER_CONFIG` but it doesn't exist rather than falling back to not using it.  Maybe something like

`if [ -n "${LINTER_CONFIG}" ] && [ ! -f "pipeline-config/${LINTER_CONFIG}" ]; then ...`